ruby/rails/security/audit/xss/avoid-default-routes.rb

```diff
@@ -0,0 +1 @@
+map.connect ":controller/:action/:id"
```

Comment:
Creating a better test is impossible for now (need some sort of functionality to ignore paths in tests)

Reply:
thats ok, but I think you need to add `ruleid: avoid-default-routes` to the test

Reply:
One way we might be able to fix this is to see if scanning for `*routes.rb` correctly picks up `config/routes.rb`, and then we could rename this test file to `avoid-default-routes.routes.rb`. That might get the test engine to pick it up.